src/pages/Regions/Region.tsx

```diff
@@ -373,7 +454,20 @@ const Region = () => {
       }
     }
     else{
-      await editRegionAdmin(admin);
+      if(admin.adminType.adminTypeName === "Голова ОПР" ||
+      admin.adminType.adminTypeName === "Член ОПР" ||
+      admin.adminType.adminTypeName === Roles.OkrugaHead ||
+      admin.adminType.adminTypeName === Roles.OkrugaHeadDeputy){
```

Comment:
Maybe, we can simplify this check. You can write all of these Roles to one array and then search only for one item from this array using .includes or sth like that